core/templates/dev/head/pages/collection_player/collection_player.html

```diff
@@ -303,6 +228,21 @@ <h2 ng-if="collectionPlaythrough.hasFinishedCollection()" class="oppia-page-head
                  version="1.1"
                  style="position: absolute; left: 50%; transform: translate(-50%, 195px);"
                  ng-click="scrollToLocation('mobile-path-anchor-' + $index); updateExplorationPreview(node.getExplorationId());">
+              <image ng-if="!collectionPlaythrough.hasStartedCollection() && $index===0"
+                     x="4"
+                     y="90"
+                     height="80"
+                     width="80"
+                     ng-attr-xlink:href="<[getStaticImageUrl('/general/collection_start_here_arrow.svg')]>" 
+                     xlink:href=""></image>
+              <text ng-if="!collectionPlaythrough.hasStartedCollection() && $index===0"
+                    x="34"
+                    y="137"
+                    alignment-baseline="middle"
+                    font-family="Capriola, Roboto, Arial, sans-serif"
+                    font-size="12"
```

Comment:
![screenshot_20170610-042432](https://user-images.githubusercontent.com/13452436/26997146-e664c74a-4d94-11e7-9bfb-34e74b945b91.png)

Is this fine? If not cannot you suggest the font?